src/develop/pixelpipe_hb.c

```diff
@@ -2517,13 +2517,9 @@ gboolean dt_dev_write_rawdetail_mask(dt_dev_pixelpipe_iop_t *piece, float *const
   p->rawdetail_mask_data = mask;
   memcpy(&p->rawdetail_mask_roi, roi_in, sizeof(dt_iop_roi_t));
 
-  dt_aligned_pixel_t wb = { piece->pipe->dsc.temperature.coeffs[0],
-                            piece->pipe->dsc.temperature.coeffs[1],
-                            piece->pipe->dsc.temperature.coeffs[2] };
-  if((p->want_detail_mask & ~DT_DEV_DETAIL_MASK_REQUIRED) == DT_DEV_DETAIL_MASK_RAWPREPARE)
```

Comment:
Why removing this? It seems related to masking... Or maybe it was some kind of workaround for the issue you've properly fixed here?

Reply:
Yes. It was a workaround.

Reply:
Noted, thanks!